src/Core/Commander/Providers/WMS_Provider.js

```diff
@@ -73,6 +58,8 @@ WMS_Provider.prototype.preprocessDataLayer = function preprocessDataLayer(layer)
     layer.version = layer.version || '1.3.0';
     layer.style = layer.style || '';
     layer.transparent = layer.transparent || false;
+    layer.options = {};
+    layer.options.tileMatrixSet = layer.tileMatrixSet || 'WGS84G';
```

Comment:
See previous comment thread https://github.com/iTowns/itowns2/pull/244#discussion_r104377515
(TL;DR: *WMS is not a tiled protocol and has nothing to do with WMTS coords*)